compose/services/docker-compose.postgis.yml

```
@@ -0,0 +1,27 @@
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+      - agwise-pg:/var/lib/postgresql/data
+    networks:
+      - internal
+    environment:
+      POSTGRES_DB: agwise_db
+      POSTGRES_PASSWORD: ${PG_PASS}
+    volumes:
+    - agwise-pg:/var/lib/postgresql/data
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)